### PR TITLE
Mod/11095 outlay toggle on state tree map

### PR DESCRIPTION
**Pull request by brianpetway**
**High level description:**

Add Outlays toggle and 'what is this?' accordion to state page Award Breakdown section, but hidden in upper environments

**Technical details:**

Copied the toggle and accordion from the Status of Funds table; left the accordion content the same, as specified in the ticket; added code for the toggle function to change the colors, as specified in the mock, but commented out that part so the colors will remain the same in Prod for now;
I stacked these components at mobile and made them change at 768, which is slightly different than how it's handled in the SoF section, but seemed right for this instance

**JIRA Ticket:**
[DEV-11095](https://federal-spending-transparency.atlassian.net/browse/DEV-11095)

**Mockup:**
two, in ticket

The following are ALL required for the PR to be merged:

Author:
- [ x] Linked to this PR in JIRA ticket
- [ ] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [ ] Verified cross-browser compatibility: Chrome, Safari, Firefox, Edge
- [ ] Verified mobile/tablet/desktop/monitor responsiveness
- [ ] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [ ] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
- [ ] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [ ] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [ ] Design review complete `if applicable`
- [ ] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [ ] Code review complete

### CODE REVIEW

File: src/js/helpers/treemapHelper.js [hunk 1]
```diff
@@ -54,13 +54,32 @@ export const tooltipStyles = {
...
-    "#053547",
-    "#325566",
-    "#587786",
-    "#7E9BA8",
-    "#A6C2CB"
+    "#053547", // direct payments
+    "#325566", // grants
+    "#587786", // contracts
+    "#7E9BA8", // other financial assistance
+    "#A6C2CB" // loans
```
Comment (anjenkin) [inline src/js/helpers/treemapHelper.js:61]: the color codes map to the amount from largest to smallest not the award type so we should remove these comments

File: src/js/helpers/treemapHelper.js [hunk 2]
```diff
@@ -54,13 +54,32 @@ export const tooltipStyles = {
...
+//     "#009ec1", // other, $cyan-40v
+//     "#00BDE3" // loans, $cyan-30v
+// ];
+//
+// export const stateTreemapColorsWithToggle = [
+//     "#0f6460", // direct payments, $mint-cool-60v
+//     "#008480", // grants, $mint-cool-50v
+//     "#00a398", // contracts, $mint-cool-40v
+//     "#1dc2ae", // other, $mint-cool-30v
+//     "#29e1cb" // loans, $mint-cool-20v
```
Comment (anjenkin) [inline src/js/helpers/treemapHelper.js:80]: same as above

**Review by anjenkin — CHANGES_REQUESTED** [review]
looks good.  see comments inline

**Comment by brianpetway** [comment]
I updated those comments

**Review by anjenkin — APPROVED** [review]

**Pull request merged**